client/lib/application/src/folded-scope-list.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import _ from 'lodash';
+import Icon from 'react-fa';
+import 'common/asset/less/application/scope-list.less';
+
+class OwnerScopeList extends React.Component {
+    constructor(props) {
+        super(props);
```

Comment:
also no deal breaker, but:
const {allResources, scopes: filtered, selected} = props;
this.state = {
            allResources,
            foldings:{},
            term: '',
            filtered,
            selected: selected || []
        };

Reply:
done